superset/migrations/versions/c25cb2c78727_security_coverage_annotations.py

```diff
@@ -0,0 +1,84 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
nit: calling these migrations "security converge <*>"

Reply:
Thanks for noticing that! Changed :)